smarty/templates/install.tpl

```diff
@@ -175,7 +175,7 @@
             <fieldset>
                 <legend>Options</legend>
             <div>
-                <div class="col-md-2">
+                <div class="col-md-9">
```

Comment:
Isn't bootstrap based on a 12 column grid?

This changes it to 9+9 = 18 columns. (Granted, 9+2=11 also wasn't right, but it's closer..)